examples/rosettacode/loops_downward_for.nit

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env nit
+#
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+
+# Task: Loops/Downward for
+#
+# SEE: http://rosettacode.org/wiki/Loops/Downward_for
+module loops_downward_for
+
+for i in [0..10].reverse_iterator do
```

Comment:
Currently [10..5] is the empty range. I'm not sure how people and mathematicians will react on begin some kind of reversed range. Created #1339